homeassistant/components/withings/common.py

```diff
@@ -85,7 +85,7 @@ class WithingsAttribute:
     """Immutable class for describing withings sensor data."""
 
     measurement: Measurement
-    measute_type: Enum
+    measure_type: NotifyAppli | GetSleepSummaryField | MeasureType
```

Comment:
Have you checked that we don't instantiate this class with keywords anywhere else?

Reply:
Yes I checked. No other references to `measute_type`